api/server/sdk/cloud_backup.go

```diff
@@ -220,14 +220,22 @@ func (s *CloudBackupServer) EnumerateWithFilters(
 		}
 	}
 
-	r, err := s.driver(ctx).CloudBackupEnumerate(&api.CloudBackupEnumerateRequest{
+	enumerateReq := &api.CloudBackupEnumerateRequest{
 		CloudBackupGenericRequest: api.CloudBackupGenericRequest{
 			SrcVolumeID:    req.GetSrcVolumeId(),
 			ClusterID:      req.GetClusterId(),
 			CredentialUUID: credId,
 			All:            req.GetAll(),
+			MetadataFilter: req.MetadataFilter,
 		},
-	})
+		ContinuationToken: req.ContinuationToken,
+		MaxBackups:        req.MaxBackups,
+	}
+	if req.StatusFilter != api.SdkCloudBackupStatusType_SdkCloudBackupStatusTypeUnknown {
+		enumerateReq.StatusFilter = api.CloudBackupStatusType(api.SdkCloudBackupStatusTypeToCloudBackupStatusString(req.StatusFilter))
```

Comment:
Can we use a bigger API name for `SdkCloudBackupStatusTypeToCloudBackupStatusString`?